modules/geo/src/internalClusterTest/java/org/opensearch/geo/GeoModulePluginIntegTestCase.java

```diff
@@ -24,6 +26,8 @@ public abstract class GeoModulePluginIntegTestCase extends OpenSearchIntegTestCa
 
     protected static final double GEOHASH_TOLERANCE = 1E-5D;
 
+    protected static final WellKnownText WKT = new WellKnownText(true, new StandardValidator(true));
```

Comment:
Any reason to prefer putting this constant here, as opposed to making it a private constant in the one file that it is used in?

Reply:
It is being used across the IT(GeoHash and GeoTile) hence I put it in the base class, and I am thinking it is going to get used for other IT also in future.